app.js

```diff
@@ -0,0 +1,123 @@
+class ToDoList {
+    constructor() {
+        this.toDoTasks = [
+            // {
+            //     id: 1,
+            //     task: 'kupic 4 bulki',
+            //     completed: false,
+            // },
+            // {
+            //     id: 2,
+            //     task: 'przedluzyc abonament',
+            //     completed: false,
+            // },11`1
+            // {
+            //     id: 3,
+            //     task: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi amet incidunt, mollitia vero doloremque quisquam corporis veritatis eum dolore commodi esse iste ratione nemo. Cumque ad eaque quo. Sapiente, perspiciatis?',
+            //     completed: true,
+            // }
+        ];
+
+        document.getElementById('addTaskBtn').addEventListener('click', this.addTask);
+        this.inputAddTask = document.getElementById('addTaskInput');
+        this.taskSection = document.querySelector('.toDoTasks');
```

Comment:
To jest akurat moja dygresja, aniżeli jakaś wielka konwencja, ale ja mam w zwyczaju używać id'ków (i wtedy `.getElementById` jak to zrobiłeś w linii 22) do elementów które w domyśle występują raz. W tym konkretnym przypadku nie ma to większego znaczenia, ale przy większej aplikacji jak się kiedyś jebniesz i zapomnisz o istnieniu drugiej sekcji o podobnej klasie.. No to wtedy jest klopsik :)

Reply:
A uwierz mi, przy dużych appkach zapomnieć o cssowej klasie ktora miała występować raz a już nie występuje zdarza się baaaardzo często :)